hack/redis/redis-load/pkg/string/randomString.go

```diff
@@ -0,0 +1,22 @@
+package string
+
+import (
+	"math/rand"
+	"time"
+)
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+var seededRand = rand.New(rand.NewSource(time.Now().UnixNano()))
+
+func stringWithCharset(length int, charset string) string {
+	b := make([]byte, length)
+	for i := range b {
+		b[i] = charset[seededRand.Intn(len(charset))]
```

Comment:
Seeing

```
panic: runtime error: index out of range

goroutine 7 [running]:
math/rand.(*rngSource).Uint64(...)
	/usr/local/go/src/math/rand/rng.go:246
math/rand.(*rngSource).Int63(0xc420091500, 0x2517525b9a1f4709)
	/usr/local/go/src/math/rand/rng.go:231 +0x8a
math/rand.(*Rand).Int63(0xc4200801e0, 0x2517525b9a1f4709)
	/usr/local/go/src/math/rand/rand.go:82 +0x33
math/rand.(*Rand).Int31(0xc4200801e0, 0x2517525b)
	/usr/local/go/src/math/rand/rand.go:96 +0x2b
math/rand.(*Rand).Int31n(0xc4200801e0, 0x3e, 0xc400000015)
	/usr/local/go/src/math/rand/rand.go:131 +0x4f
math/rand.(*Rand).Intn(0xc4200801e0, 0x3e, 0x15)
	/usr/local/go/src/math/rand/rand.go:169 +0x45
github.com/ciaranRoche/redis-load/pkg/random.stringWithCharset(0x2a, 0x619245, 0x3e, 0x0, 0x0)
	/go/src/github.com/ciaranRoche/redis-load/pkg/random/randomString.go:15 +0xab
github.com/ciaranRoche/redis-load/pkg/random.RandomString(0x2a, 0x60e2dc, 0x5)
	/go/src/github.com/ciaranRoche/redis-load/pkg/random/randomString.go:21 +0x40
main.main.func1()
	/go/src/github.com/ciaranRoche/redis-load/main.go:44 +0x42f
github.com/ciaranRoche/redis-load/pkg/concurrency.ConcurrentFuncs.func1(0xc420016430, 0xc420010710)
	/go/src/github.com/ciaranRoche/redis-load/pkg/concurrency/concurrent.go:19 +0x4d
created by github.com/ciaranRoche/redis-load/pkg/concurrency.ConcurrentFuncs
	/go/src/github.com/ciaranRoche/redis-load/pkg/concurrency/concurrent.go:17 +0x20f
```

Reply:
I think i have fixed this, after testing with 1 million entries over 10 users, I am hitting 
```time="2020-08-06T10:49:41Z" level=fatal msg="OOM command not allowed when used memory > 'maxmemory'." action=makeNoise
```
essentially I reduced the charset to use a reduced number of bytes.